addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4961,7 +4969,7 @@ msgstr ""
 
 #: unknown
 msgctxt "#12362"
-msgid "Shutdown system if Master Lock retries exceeded"
+msgid "Shutdown system if \"Master lock\" retries exceeded"
```

Comment:
"Shutdown system if number of master lock retries is exceeded."
It's the number of retries :)